build.sbt

```diff
@@ -14,7 +14,7 @@ val circeVersion: Option[(Long, Long)] => String = {
 val sttpVersion = "3.3.15"
 val Specs2Version = "4.2.0"
 val artifactory = "https://cognite.jfrog.io/cognite/"
-val cogniteSdkVersion = "1.5.16"
+val cogniteSdkVersion = "1.5.18-SNAPSHOT"
```

Comment:
TODO: Update this once https://github.com/cognitedata/cognite-sdk-scala/pull/401 is merged

Reply:
```suggestion
val cogniteSdkVersion = "1.5.18"
```